README.rst

```diff
@@ -253,6 +279,49 @@ in terms of disk space.
 
 Whether ping checks are created automatically for devices.
 
+``OPENWISP_MONITORING_AUTO_DEVICE_CONFIG_CHECK``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-------------+
+| **type**:    | ``bool``    |
++--------------+-------------+
+| **default**: | ``True``    |
++--------------+-------------+
+
+This setting allows you to choose whether `config_modified <#Configuration>` checks should be
+created automatically for newly registered devices. It's enabled by default.
+
+``OPENWISP_MONITORING_DEVICE_CONFIG_CHECK_MAX_TIME``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-----------+
+| **type**:    |  ``int``  |
++--------------+-----------+
+| **default**: |   ``5``   |
++--------------+-----------+
+
+``config_modified`` check changes the device health status to ``PROBLEM`` if it's
+configuration status has been in the modified state for more than  the time defined in
+``OPENWISP_MONITORING_AUTO_DEVICE_CONFIG_CHECK``. It's 5 minutes by default.
+
+**Note**: If the setting ``OPENWISP_MONITORING_AUTO_DEVICE_CONFIG_CHECK``
+is disabled then this setting need not be declared.
```

Comment:
"This setting will be ignored if `OPENWISP_MONITORING_AUTO_DEVICE_CONFIG_CHECK = False`"